pipeline/stage02_processing/scripts/spatial_downsampling.py

```diff
@@ -0,0 +1,89 @@
+"""
+Substract the background of a given dataset by substracting the mean of each channel.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+import argparse
+import os
+import quantities as pq
+from skimage import data, io, filters, measure
+from utils import determine_spatial_scale, load_neo, write_neo, save_plot, \
+                  none_or_str
+
+
+def spatial_smoothing(asig, MACRO_PIXEL_DIM, DIM_X, DIM_Y):
+
+    print("         Spacial smoothing...")
```

Comment:
The logging and screen output is done by snakemake, so there is no need to print these state information within the scripts or functions